resources/integration_tests.rs.in

```diff
@@ -113,9 +87,23 @@ mod tests {
         let mut builder = InMemoryWasmTestBuilder::default();
         builder
             .run_genesis(&PRODUCTION_RUN_GENESIS_REQUEST)
-            .exec(execute_request)
-            .commit()
-            .expect_failure();
+            .commit();
+        builder.exec(execute_request).commit().expect_failure();
+
+        let error = ApiError::MissingArgument;
+        let error_code: u32 = error.into();
+        let actual_error = builder.get_error().expect("must have error");
+        let reason = "should error on missing runtime_arg";
+        let actual = format!("{actual_error:?}");
+        let expected = format!(
+            "{:?}",
+            EngineStateError::Exec(execution::Error::Revert(error))
+        );
+
+        assert_eq!(
+            actual, expected,
+            "Error should match {error_code} with reason: {reason}"
+        )
```

Comment:
How about this form? Most if not all integration tests are written this way:

```rust
assert!(matches!(
    actual_error,
    EngineStateError::Exec(execution::Error::Revert(api_error))
    if api_error == ApiError::MissingArgument.into()),
    "Expected missing argument, received {actual_error:?}");
```

Reply:
Well personally I find this form really hard to read, specifically the if statement that make it looks like a typo.

Changed it as per reco.

Reply:
@mpapierski @Fraser999 I made a last clean up https://github.com/casper-ecosystem/cargo-casper/pull/50/commits/0d3efaf395fb093c7968f4b65a97921acd8f33eb please check it looks correct.

Reply:
https://github.com/casper-ecosystem/cargo-casper/pull/50/commits/c239cea11f13116c7f8ef74aed8e2556fec330b1